include/deal.II/fe/fe_interface_values.h

```diff
@@ -551,6 +581,12 @@ FEInterfaceValues<dim, spacedim>::reinit(
       fe_face_values_neighbor = &internal_fe_subface_values_neighbor;
     }
 
+  AssertDimension(fe_face_values->n_quadrature_points,
+                  fe_face_values_neighbor->n_quadrature_points);
+
+  const_cast<unsigned int &>(this->n_quadrature_points) =
```

Comment:
Could we make `FEInterfaceValues::n_quadrature_points` non-const?

Reply:
Different reviewers different opinions: https://github.com/dealii/dealii/pull/10764#discussion_r541937876

Reply:
Lets keep it this way for now but we should eventually clean this up.